app/views/order_items/_order_item.html.erb

```diff
@@ -5,6 +5,17 @@
         <%= image_tag product.avatar %>
       </div>
       <div class="caption">
+        <h6>
+          <% if product.calories != nil %>
+            <%= content_tag( :small) do
+              concat( product.calories)
+              concat ' kcal'
+              end
+            %>
+          <% else %>
```

Comment:
ok



On 8 Jul 2015 at 13:24:40, VandorpeDavid (notifications@github.com) wrote:

In app/views/order_items/_order_item.html.erb:

> @@ -5,6 +5,17 @@
>          <%= image_tag product.avatar %>
>        </div>
>        <div class="caption">
> +        <h6>
> +          <% if product.calories != nil %>
> +            <%= content_tag( :small) do
> +              concat( product.calories)
> +              concat ' kcal'
> +              end
> +            %>
> +          <% else %>
@Silox Result when removing the   http://screencloud.net/v/bl3V

—
Reply to this email directly or view it on GitHub.